run

```diff
@@ -156,6 +156,8 @@ sub get_platform_name {
     if ($platform eq "tyan,palmetto" ||
         $platform eq "tyan,habanero" ||
         $platform eq "ibm,firestone" ||
+        $platform eq "ibm,firenze"   ||
+        $platform eq "ibm,powernv"   ||
         $platform eq "ibm,garrison")
```

Comment:
@stewart-ibm  For users who don't know the platform type they can start use ibm,powernv generic platform type.